packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -253,7 +253,8 @@ export function TransferPanelSummary({ token }: TransferPanelSummaryProps) {
           {isBatchTransferSupported && Number(amount2) > 0 && (
             <span>
               {' '}
-              and {amount2} {childChainNativeCurrency.symbol}
+              and {formatAmount(Number(amount2))}{' '}
```

Comment:
`formatAmount` accept `BigNumber | undefined`, we should not need to cast here. Also, casting to number directly might cause issues with overflow

Reply:
`amount` and `amount2` are strings from query params. We've been casting it like this before so I'm not introducing any new changes, maybe here also we can make a ticket to refactor to use bignumbers?